McKenna Adam model

marcdotson opened the pull request:
Hey, @marcdotson take a look.

marcdotson commented:
Hey, @weechm @adamhettinger1 I see two `Model beg.R` files. Will you merge them? Or get rid of the one you're not using?

weechm commented:
@marcdotson we made a few adjustments to the model if you want to take a look. One of the things we were playing with was trying to scale the data so binary variables like past would have similar effects with variables like followers. We were also looking at trying to manipulate the data to force some type of correlation between the variables but we were not successful

marcdotson commented:
@weechm @adamhettinger1 I've cleaned up your code and renamed some things (including the name of the R script). `quap()` runs fine. If you just print the output object you can see that the posterior means for each of the parameters is close to the true values used in simulating the data.

Here's what I want from you for Friday.

1. Use this example, including the `quap()` function, to teach me about Bayesian inference.
2. Come prepared to explain and discuss the ecology model as specified in the paper, especially how it differs from the simple model used in this example.
3. I don't know if I just gave this assignment to @weechm or both of you, but if you've found any helpful introductions to predictive modeling, let's discuss those as well.

I'll merge this pull request on Friday as well, so anything you want to add beyond printing the model output, do it before then and we can discuss. Also, I'm closing #30 because it's a bad idea. We can talk about why that is on Friday as well.
